libraries/lib-wave-track/WaveTrack.cpp

```diff
@@ -1987,6 +1986,24 @@ bool WaveTrack::CloseLock() noexcept
    return true;
 }
 
+ClipConstHolders WaveTrack::GetClipInterfaces() const
+{
+   assert(IsLeader());
+   const auto pOwner = GetOwner();
+   ClipConstHolders wideClips;
+   wideClips.reserve(mClips.size());
+   for (auto clipIndex = 0u; clipIndex < mClips.size(); ++clipIndex)
+   {
+      const auto leftClip = mClips[clipIndex];
+      const auto rightClip =
+         NChannels() == 2u && pOwner ?
+            (*++pOwner->Find<const WaveTrack>(this))->mClips[clipIndex] :
+            nullptr;
+      wideClips.emplace_back(std::make_shared<WideClip>(leftClip, rightClip));
```

Comment:
The informally stated (and not `assert`ed) precondition of the constructor of `WaveClip` can be violated if you make a track with misaligned clips.

But of course it doesn't matter much.  WideClip is provisional only until I make real wide WaveClips and misaligned clips become impossible.